### PR TITLE
Doc cpex0043

### PR DESCRIPTION
**Pull request by poinot**
Doc modifs for CPEX0043 Family hierachy tree.

Modifs as described in the CPEX proposal (SIDS, FMM, MLL)

Mostly copy/paste from existing doc of the Family_t

### CODE REVIEW

File: CGNS_docs_current/sids/cgnsbase.html [hunk 1]
```diff
@@ -371,12 +371,20 @@ <h4>6.2.1 Base Level Families</h4>
 name of the <tt>Family_t</tt> has to be prefixed by the <tt>CGNSBase_t</tt> name.
 The pattern is then <tt>basename/familyname</tt>, only one single /
 character is allowed, and neither of <tt>basename</tt> nor <tt>familyname</tt>
-should be empty.
+should be empty. This <tt>Family_t</tt> node can be a direct child of the
+<tt>CGNSBase_t</tt> or a child of another <tt>Family_t</tt>. The actual family name
+has the pattern
+<tt>&lt;CGNSBase_t&gt;/&lt;FamilyName1&gt;/&lt;FamilyName2&gt;/.../&lt;FamilyNameN&gt;</tt>.
```
Comment (MicK7) [inline CGNS_docs_current/sids/cgnsbase.html]: missing / before CGNSBase_t ?

File: CGNS_docs_current/sids/cgnsbase.html [hunk 2]
```diff
@@ -371,12 +371,20 @@ <h4>6.2.1 Base Level Families</h4>
...
 The family-mesh association is defined under the
-<a href="#Zone"><tt>Zone_t</tt></a> and
+<a href="#Zone"><tt>Zone_t</tt></a>,  <a href="#ZoneSubRegion"><tt>ZoneSubRegion_t</tt></a> and
 <a href="bc.html#BC"><tt>BC_t</tt></a> data structures by specifying
-the family name corresponding to a zone or a boundary patch.
+the family name corresponding to a zone, a zone sub-region or a boundary patch in a
+<tt>FamilyName_t</tt> node. If the value of the <tt>FamilyName_t</tt> node does not have
+a <tt>/</tt> character in it, then the name refers to a family being a direct child of its
+CGNS base. Otherwise, if this value has at least one <tt>/</tt> in it, the pattern
+<tt>&lt;CGNSBase_t&gt;/&lt;FamilyName1&gt;/&lt;FamilyName2&gt;/.../&lt;FamilyNameN&gt;</tt>
```
Comment (MicK7) [inline CGNS_docs_current/sids/cgnsbase.html]: missing / before CGNSBase_t ?

**Review by MicK7 — APPROVED** [review]
Everything is in place (just check my comments on / before CGNSBase_t)

File: CGNS_docs_current/sids/cgnsbase.html (hunk 2 above)
Comment (poinot) [reply CGNS_docs_current/sids/cgnsbase.html]: done

**Pull request merged**